src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -57,6 +59,20 @@ class PipelineInfo:
     failures: str
 
 
+@dataclass
+class AzureServicePrincipalInfo:
+    # fs.azure.account.oauth2.client.id
+    application_id: str
+    # fs.azure.account.oauth2.client.secret: {{secrets/${local.secret_scope}/${local.secret_key}}}
+    secret_scope: str
+    # fs.azure.account.oauth2.client.secret: {{secrets/${local.secret_scope}/${local.secret_key}}}
+    secret_key: str
+    # fs.azure.account.oauth2.client.endpoint: "https://login.microsoftonline.com/${local.tenant_id}/oauth2/token"
+    tenant_id: str
```

Comment:
is this necessary to store every time? The tenant should be the same if they're in the same account no?

Reply:
with this data model it's easier to duplicate